lib/useKWindowDimensions.js

```diff
@@ -1,5 +1,6 @@
 import './composition-api'; //Due to @vue/composition-api shortcomings, add plugin prior to use in kolibri, studio and tests
 import { onMounted, onUnmounted, ref } from '@vue/composition-api';
+import debounce from 'lodash/debounce';
```

Comment:
Just FYI, [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) might be appropriate here because it implements a dynamic throttle. The old responsive window mixin used the `frame-throttle` wrapper library for this:

https://github.com/learningequality/kolibri-design-system/blob/e578edef93471f78c051599a8162043710ad8c28/lib/KResponsiveWindowMixin.js#L59

https://github.com/learningequality/kolibri-design-system/blob/e578edef93471f78c051599a8162043710ad8c28/lib/KResponsiveWindowMixin.js#L74-L77

Reply:
Hello @indirectlylit, Thanks for your review. Just to confirm, do I need to implement it similar to how it is implemented in `KResponsiveWindowMixin.js` using `throttle`?

Reply:
Hi @Jaspreet-singh-1032 - that's for you (and @MisRob) to decide. This is a _very_ minor note and should not block the PR.

That said if you're interested, some links:

* https://css-tricks.com/using-requestanimationframe/
* https://css-tricks.com/debouncing-throttling-explained-examples/

Personally, I prefer using this form of throttling for anything related to fluid UI changes because the system can choose an appropriate refresh rate rather than hard-coding a somewhat arbitrary delay period.

Reply:
Thank you, @indirectlylit, I was previously unaware of this function. I will read about it and If it is a better option then I will definitely use it here.

Reply:
For most browsers this will use requestAnimationFrame, which will invoke the callback just before the next repaint - which means that it should not be the cause of a reflow by reading the inner width and height of the window (which is a strong part of the motivation for this issue), so I do think this is a good change to use here.

Reply:
Thanks, @rtibbles, I read some blogs about `requestAnimationFrame` and I also think this is a better option to use here.

Reply:
Thanks @indirectlylit, that's a good idea